README.md

```diff
@@ -14,32 +14,35 @@ If you want to get data from the chain you tend to have to use a provider like i
 
 If you wish to build a big data lake or even just fetch dynamic events from the chain this task is near impossible without a third party paid tool or something like thegraph hoster services and most do not let you pull in millions of rows at a time fast. This data should be able to be fetched for free, blazing fast and customisable to your needs. This is what reth-indexer does.
 
-This project aims to solve this by reading directly from the reth node db and indexing the data into a postgres database you can then query fast with indexes already applied for you. You can also scale this easily by running multiple instances of this program on different boxes and pointing them at the same postgres database (we should build that in the tool directly).
+This project aims to solve this by reading directly from the reth node db and indexing the data into traditional databases that you can then query fast with indexes already applied for you. You can also scale this easily by running multiple instances of this program on different boxes and pointing them at the same underlying database (we should build that in the tool directly).
 
 This tool is perfect for all kinds of people from developers, to data anaylsis, to ML developers to anyone who wants to just get a snapshot of event data and use it in a production app or just for their own reports.
 
 ## Features
 
-- Creates postgres tables for you automatically
+- Creates database tables for you automatically (postgres, gcp biquery, or others)
 - Creates indexes on the tables for you automatically to allow you to query the data fast
+- Can write index to multiple data stores (i.e. postgres, gcp bigquery, etc)
 - Indexes any events from the reth node db
 - Supports indexing any events from multiple contracts or all contracts at the same time
 - Supports filtering even on the single input types so allowing you to filter on every element of the event
 - Snapshot between from and to block numbers
 - No code required it is all driven by a json config file that is easy to edit and understand
-- Created on your own infurstructure so you can scale it as you wish
-- Exposes a ready to go API for you to query the data
+- Created on your own infrastructure so you can scale it as you wish
+- Exposes a ready to go API for you to query the data (postgres implementation only at this time)
+- Abstractions to allow for extension of this indexer to different databases / datastores
 
 ## Benchmarks
 
-Very hard to benchmark as it is all down to the block range and how often your event is emitted but roughly: (most likely could be speed up with more some optimisations.)
+Very hard to benchmark as it is all down to the block range and how often your event is emitted but roughly: (most likely could be speed up with more some optimisations).  Note this applies only to the postgres database write.
```

Comment:
is there no way you can run the same benchmark on your machine to see the numbers for public big query? as its free to write this would be really good the benchmark config is checked in - no worries if not

Reply:
i think that's reasonable.  the issue is that i don't have that many blocks synced up on my local reth node (thought i had up to 18m but its much smaller i.e. 6m or so), won't have sufficient space on my machine to get up to the range of the benchmarks though - is there some way i could sync up to the blocks that i have and then extrapolate the performance?  i.e. if up to 6m, then would that be close enough for the range of the existing benchmark?  i.e. 11m..17m is approx 6m?  not sure if the block sizes / # of transactions increased over time for later blocks.

Reply:
cool i think its fine if we add benchmarks later